kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java

```diff
@@ -49,50 +49,17 @@
 import org.dashbuilder.json.JsonObject;
 import org.dashbuilder.json.JsonType;
 import org.dashbuilder.json.JsonValue;
+import static org.dashbuilder.dataset.json.DataSetLookupJSONMarshallerConstants.*;
 
 public class DataSetLookupJSONMarshaller {
 
-    private static final String UUID = "dataSetUuid";
```

Comment:
@bxf12315 I don't see much point in this refactor as these constants are private anyway.

Reply:
Unit test need some the field, so I refactor it.

Reply:
you could just make the needed constants protected?